pytorch_blade/src/ltc/disc_compiler/replay.cpp

```diff
@@ -106,18 +130,16 @@ void LoadAndReplay(const std::string& path, int iters, int warmup) {
       }
     }
   }
-}
-
-void FusionOutputs(std::shared_ptr<torch::jit::Graph> graph) {
-  auto return_node = graph->return_node();
-  if (return_node->inputs().size() <= 1) {
-    return;
+  {
+#ifdef TORCH_BLADE_BUILD_WITH_CUDA
+    cudaProfilerStart();
```

Comment:
How about adding a flag here, such as TORCH_DISC_ENABLE_NVPROF_EXCLUSIVE=on?

Reply:
The replay toolkit is used for profiling, for my personal view, it's convenient to enable nvprofile API by default, and do not need a new env var.